litgpt/tokenizer.py

```diff
@@ -54,6 +54,14 @@ def __init__(self, checkpoint_dir: Union[Path, str]) -> None:
         else:
             raise NotImplementedError
 
+        # NOTE: A temporary fix until it's resolved on Tokenizers side.
```

Comment:
FIY, affects also SentencePiece. So something is wrong with vocab files, I suppose.

Reply:
Hm, that's interesting

Reply:
Yep interesting indeed.
For some models sentencepiece doesn't have this problem, but for others, like open_llama that is shipped only with SP vocab, we have this issue.
It just wasn't checked before.